src/Autosuggest.js

```diff
@@ -440,6 +444,7 @@ class Autosuggest extends Component {
         inputProps={autowhateverInputProps}
         itemProps={this.itemProps}
         theme={theme}
+        inputElement={this.props.inputElement}
```

Comment:
If I understand correctly, we'll be passing `undefined` here by default, relying on `react-autowhatever` to have an `'input'` fallback.

What do you think about being more explicit by:
* making `inputElement ` required in `Autosuggest.js`
* default `inputElement` to `'input'` in `AutosuggestContainer.js`

?